src/MaterialEditor.Core.Studio/Core.MaterialEditor.SceneController.cs

```diff
@@ -589,15 +620,18 @@ protected override void OnObjectDeleted(ObjectCtrlInfo objectCtrlInfo)
 
         protected override void OnObjectVisibilityToggled(ObjectCtrlInfo objectCtrlInfo, bool visible)
         {
-            if (visible && objectCtrlInfo is OCIItem item)
+            if (objectCtrlInfo is OCIItem item)
             {
                 var id = item.GetSceneId();
-                foreach (var property in RendererPropertyList.Where(x => x.ID == id && x.Property == RendererProperties.Enabled))
-                {
-                    MaterialAPI.SetRendererProperty(GetObjectByID(id), property.RendererName, property.Property, property.Value);
-                    // potential recalc of normals, have to test...
-                }
+                if(visible)
```

Comment:
Why change this?

Reply:
oops. First changed it to match a projectors enabled state with the items visible state. But I removed it to not mess anything up for those games that already had projectors and may have enabled items, but disabled projectors.

Will change it back to its original state